spyder/api/plugins.py

```diff
@@ -31,6 +31,10 @@ class BasePlugin(BasePluginMixin):
 
     WARNING: Don't override any methods or attributes present here!
     """
+    # Define configuration name map for plugin to split configuration
+    # among several files. See spyder/config/main.py
+    # Status: Hidden
+    _CONF_NAME_MAP = None
```

Comment:
@ccordoba12 since the CONF_MAP option is advanced, I will hide it for now from the API.

Reply:
Ok, no problem. Then you should move it to `BasePluginMixin` until we decide expose it here.

Reply:
👍